packages/manager/apps/container/src/__tests__/navbar/__snapshots__/navbar.spec.js.snap

```diff
@@ -67,7 +67,7 @@ exports[`UI testing of the navbar renders correctly with environment and shell 1
             aria-expanded="false"
             aria-haspopup="false"
             aria-label="language_change"
-            class="oui-navbar-link oui-navbar-link_dropdown"
+            class="oui-navbar-link oui-navbar-link_dropdown "
```

Comment:
Uhm, the snap didn't really change, isn't just a space added in the code that should be removed instead?

Reply:
```diff
-             class="oui-navbar-link oui-navbar-link_dropdown"
+             class="oui-navbar-link oui-navbar-link_dropdown "
```

It was the issue reported by https://github.com/ovh/manager/runs/6370955780?check_suite_focus=true